website/rss.py

```diff
@@ -0,0 +1,18 @@
+import feedparser
+import eventlet
+
+
+with eventlet.Timeout(10):
+    url = 'http://feeds.feedburner.com/PythonInsider/.rss'
+
+
+def get_rss():
+
+    feed = feedparser.parse(url)
```

Comment:
I think this ```url``` should also be retrieved by get_url function.
You can have a thought on it.

Reply:
I don't thinks so, I have checked for 404 error. It works fine.

Reply:
I have updated the code, but I have checked for all possibilities it wont through any error or exception even the url will be empty or wrong.